css/styles.css

```diff
@@ -1,3 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
 body {
+  background-color: black;
+  color: white;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   font-size: 62.5%; /* Causes base font size to be ~10px */
+  font-family: Poppins, monospace;
```

Comment:
Id use `font-family: 'Poppins', sans-serif;` instead. A sans-serif would be a better falllback than a monospace font.